backbone/src/main/java/org/researchstack/backbone/model/SchedulesAndTasksModel.java

```diff
@@ -15,6 +15,7 @@
         public String delay;
         public String scheduleString;
         public Date scheduledOn;
+        public Date expiresOn;
```

Comment:
It seems at this point it would be valuable to focus on an easy way for SchedulesAndTasksModel to be sub-classed so we could have BridgeSchedulesAndTasksModel.  I'm sure other devs will need to add fields as well, but don't have access to the source.

Reply:
I agree that'd be good. ResearchSuite aims to clean things up and make models easily extensible.